src/utils/helpers.js

```diff
@@ -0,0 +1,162 @@
+function get(obj, path, defaultValue) {
+  const keys = path.split('.');
+  let result = obj;
+
+  for (let key of keys) {
+    result = result[key];
+
+    if (typeof result === 'undefined') {
+      return defaultValue;
+    }
+  }
+
+  return result;
+}
+
+function identity(param) {
+  return param;
+}
+
+function first(list) {
+  return Array.isArray(list) && list.length ? list[0] : undefined;
+}
+
+function last(list) {
+  return Array.isArray(list) ? list[list.length - 1] : undefined;
+}
+
+
+
+
+const baseRange = (start, end, step, isRight) => {
+  let index = -1;
+  let length = Math.max(Math.ceil((end - start) / (step || 1)), 0);
```

Comment:
```suggestion
const baseRange = (start, end, step = 1, isRight) => {
  let index = -1;
  let length = Math.max(Math.ceil((end - start) / step), 0);
```
Значение по умолчанию лучше вынести сюда, т.к. дальше в цикле тоже используется переменная `step` и если она не определена, то результат выполнения будет не тем, что мы ожидаем

Reply:
Имя аргумента `isRight` не совсем очевидное. В данном случае его лучше назвать как-то вроде `hasReverseOrder` или что-то подобное

Reply:
Argument renamed